pygfunction/properties.py

```diff
@@ -0,0 +1,122 @@
+from __future__ import absolute_import, division, print_function
+
+from CoolProp.CoolProp import PropsSI
+import warnings
+
+
+class Fluid:
+    """
+        An object for handling the fluid properties
+
+        Parameters
+        ----------
+        m_flow: float
+            The fluid mass flow rate per borehole (kg/s)
+        T: float, optional
+            The temperature of the fluid (Kelvin)
```

Comment:
With the implementation of the `Fluid` class, we should settle on a unit system convention.

My preference is : degC, J, kg, Pa, s. This decision should be added in the _Styleguide_ section of [CONTRIBUTING.md](https://github.com/MassimoCimmino/pygfunction/blob/master/CONTRIBUTING.md).

Reply:
Mass flow rate has been removed because this object is going to be used as a method for quick access to fluid properties, not an object to be passed around inside of `pygfunction`.

The temperature passed into the function is now in Celsius.